Common/src/main/resources/assets/hexcasting/lang/zh_cn.flatten.json5

```diff
@@ -997,636 +1233,795 @@
+        "5": "每次写入都需要栈上有一个实体和一个由图案构成的列表。这个实体必须是一个含有$(media)媒质/$的物品实体（也即掉落在地上的$(l:items/amethyst)$(item)紫水晶/$）。该实体会变成该魔法物品的内部电池。$(br2)似乎电池中的$(media)媒质/$不会像用$(l:items/staff)$(item)法杖/$施法那样以物品组为单位消耗。其中的$(media)媒质/$实际上会“溶解”入一个连续的能量池。因此，当物品中存有会消耗 1 个$(l:items/amethyst)$(item)紫水晶粉/$中媒质来施放的$(hex)咒术/$时，含有相当于 1 个$(l:items/amethyst)$(item)充能紫水晶/$中媒质的电池能支持施放该咒术 10 次。",
+        "crafting.desc": "$(italic)我们常说一句话，“魔法从不……”。它从不“就是行得通”，它不会随你操控。你不能光靠所谓魔法扔个火球，烧顿晚饭，或是把一帮劫匪变成青蛙和蜗牛。/$",
+      },
 
-      "scroll.1": "$(l:items/scroll)$(item)卷轴/$能让分享图案画法变得十分方便。可用$(l:patterns/readwrite#hexcasting:write)$(action)书吏之策略/$将图案复制上去，然后该图案就会在物品提示栏内出现。$(br2)也可以将其放在墙上用作装饰或用于教学。就和画一样，卷轴也分大小，从 1x1 到 3x3 都有。对挂在墙上的卷轴使用$(l:items/amethyst)$(item)紫水晶粉/$就能显示绘制的笔画顺序。",
-      "scroll.2": "此外，我还在地牢和要塞里发现了所谓$(l:items/scroll)$(item)远古卷轴/$。它们记载了一些特殊法术的笔画顺序——$(thing)卓越法术/$，传闻是强大到凡人的意识无法承受的法术……$(br2)但如果那些“凡人”施放不了，那我觉得他们就不应该知道这些法术的存在。",
-      "scroll.crafting.desc": "$(italic)我用锐利的笔和学生的鲜血在洁白的纸上书写，探知着他们的秘密。/$",
+      phials: {
+        "1": "我觉得……自然不愿意给我的研究留些余量的做法，令人非常不快。如果我手头只有$(l:items/amethyst)$(item)充能紫水晶/$，就算是最为基础的$(l:patterns/basics#hexcasting:raycast)$(action)弓箭手之馏化/$也会消耗一整块水晶，没利用的$(media)媒质/$就被浪费了。$(br2)幸运的是，我找到了一种解决问题的方法。",
+        "2": "我找到了记载着一种注入了$(media)媒质/$的$(item)玻璃瓶/$的卷轴。施放$(hex)咒术/$时，法术会从这种试剂瓶中汲取$(media)媒质/$。液态的$(media)媒质/$便于按量使用，不会有媒质被浪费。它就和$(l:items/hexcasting)$(item)缀品/$的内部电池差不多，也一样可以用$(l:patterns/spells/hexcasting#hexcasting:recharge)$(action)重新充能/$补充媒质。",
+        "3": "而不幸的是，用于$(italic)制作/$这种试剂瓶的技艺已经散佚了。我确实发现了$(l:patterns/great_spells/make_battery#hexcasting:craft/battery)$(thing)对应图案画法的提示/$，但具体绘制方法难以捉摸，怎么都画不对。不过我想只要勤学苦练就能修得真功。现在的话，浪费$(media)媒质/$是没法避免了……$(br2)但我不会一直囿于此处的。",
+        desc: "$(italic)玉液琼浆。/$",
+      },
 
+      pigments: {
+        "1": "咒法学的古代研究者们偶尔会用一种颜色来象征自己和自己的$(hex)咒术/$。他们的名字已不可考，但那些颜色留存至今。倘若直接向自然呈上一种特殊的染色剂，就能“……以自然乐见的方式为某人的思维涂上颜色，并使个人代表色产生奇妙的变化。”",
+        "2": "我不懂它们的具体原理，但我相信我已琢磨出制造不同颜色染色剂的配方了。若要使用染色剂，可手持之并以另一只手施放$(l:patterns/spells/colorize)$(action)内化染色剂/$，染色剂则会被消耗。$(br2)染色剂似乎会影响我施放$(hex)咒术/$时的$(media)媒质/$火花以及我的$(l:patterns/spells/sentinels)$(thing)哨卫/$的颜色，但影响到其他地方也完全是正常现象。",
 
-      "slate.1": "$(l:items/slate)$(item)石板/$和$(l:items/scroll)$(item)卷轴/$类似，可以将图案复制上去并放置在世界中以展示该图案。$(br2)然而，我曾读到过某些粗略的故事，它们记述了一种用$(l:items/slate)$(item)石板/$组成的伟大结构，用来施放$(l:items/staff)$(item)法杖/$无法施放的$(l:greatwork/spellcircles)$(thing)伟大仪式/$。",
-      "slate.2": "也许我将来就会碰到这些知识。但就现在而言，我觉得它们非常适合装饰。$(br2)至少它们可被放在方块的任何一面，这点和$(l:items/scroll)$(item)卷轴/$不同。",
-      "slate.crafting.desc": "$(italic)这就是横竖撇捺了。要记牢。/$",
-      "slate.3": "我还发现了其他类型的$(l:items/slate)$(item)石板/$，它们并未刻有图案，但似乎内嵌有某种……奇怪的……异常构造。单单思考它们就让我头脑胀痛，就好像我的思维被它们的结构折弯，随着它们的回路流动，迷失在它们迷宫般的深渊中，迷失在迷失在迷失在导向到处理为——$(br2)……我的思维差点就被搅乱了。还是之后再做相关研究吧。",
+        "colored.crafting.header": "色彩染色剂",
+        "colored.crafting.desc": "五彩缤纷的染色剂。",
 
+        speical: "最后还有两种特殊的染色剂。$(item)灵魂闪光染色剂/$的颜色为我所独有，而$(item)空无染色剂/$则会重置为初始状态的紫橙混色。$(br2)$(italic)以及许多还没被研究出来的内在颜色。/$",

```

Comment:
This should be `special` instead of `speical`

Reply:
Fixed